tls-tester/pom.xml

```diff
@@ -15,7 +15,7 @@
       <plugin>
         <groupId>com.axonivy.ivy.ci</groupId>
         <artifactId>project-build-plugin</artifactId>
-        <version>10.0.14</version>
+        <version>11.3.0</version>
```

Comment:
I think the build is yet no happy with this change: you will need either
- a released version of the plugin; so you could use 11.2.0 (but then I'd also change the version of the ivy-tls-tester accordingly)
- or you use a snapshot of 11.3 > `11.3.0-SNAPSHOT` ... then you'll also need an entry to consume snapshot releases in your pom. See https://axonivy.github.io/project-build-plugin/snapshot/11.2/examples.html